app-text/asciidoc/asciidoc-9.0.0_rc1.ebuild

```diff
@@ -35,6 +35,7 @@ RDEPEND="
 DEPEND="
 	test? (
 		app-text/dvipng
+		app-text/dvisvgm
```

Comment:
Please squash this into the original version bump.

Reply:
I disagree, it's not directly relevant to the version bump (the dependency was always missing).